src/test/ui/simd-ffi-aarch64.rs

```diff
@@ -0,0 +1,57 @@
+// ignore-tidy-linelength
```

Comment:
Please move all tests that relate to `simd_ffi` into: `ui/rfc-2574-simd-ffi/$file`.